.github/actions/make_init/action.yml

```diff
@@ -50,7 +50,7 @@ runs:
     # Actions moves to a newer version of Ubunutu.
     - name: Add vendored `protoc` to $PATH
       run: |
-        echo "./vendor/protoc-3.20.3-linux-x86_64/bin" >> $GITHUB_PATH
+        echo "./vendor/protoc-26.1-linux-x86_64/bin" >> $GITHUB_PATH
```

Comment:
Was the decision to include `protoc` in our repo made to make the GitHub CI more stable and don't depend on too many external factors? I think that might make sense although we have to install a lot of external dependencies, so I am curious

Reply:
I believe the decision to vendor it in was made because the `apt-get` version you can install on the ubuntu version used by our CI is quite old (see the comment above). I double checked this and the CI would install a protoc < 3.20 version.

Reply:
hmhm good question, if everything works then probably not needed 🙂 I just checked real quick the official installation instructions but did not find anything, so probably a legacy artifact

Reply:
The includes where needed. Something did fail in the CI so I added it back again